bokehjs/src/coffee/models/glyphs/patches.coffee

```diff
@@ -77,8 +77,8 @@ class PatchesView extends Glyph.View
   _render: (ctx, indices, {sxs, sys}) ->
     # @sxss and @syss are used by _hit_point and sxc, syc
     # This is the earliest we can build them, and only build them once
```

Comment:
That comment is old. I agree with you it doesn't make sense. I do remember moving those lines around a bunch when I made this PR. This was also the PR where I made all the patches disappear if I recall correctly, so there may have been a hasty hotfix.